script.js

```diff
@@ -0,0 +1,36 @@
+const containerElement = document.querySelector('.container');
+
+// Title
+const titleElement = document.createElement('h1');
+titleElement.classList.add('title');
+titleElement.innerHTML = 'Memory Game';
+
+containerElement.appendChild(titleElement);
+
+// Card Elements
+const cardElement = document.createElement('div');
+cardElement.classList.add('card');
+containerElement.appendChild(cardElement);
+
+
+cardElement.innerHTML = `
+  <div class="card-inner">
+    <div class="card-back">
+      <img src="assets/card-backside.jpg" alt="Back of the card">
```

Comment:
You can consider uploading your pictures in github and use it as your host.
https://medium.com/@jeftachibiya360/how-to-host-images-for-your-website-on-github-a98d917284c5

Reply:
Thank you for your suggestion! I already uploaded the pictures to the repository in a folder called assets. However, I will consider using their URLs instead of the local paths from now on.

Reply:
Yes, and having them in assets folder is also good.
That is only a suggestion and good to have